docs/man/borg.1

```diff
@@ -484,6 +484,11 @@ You might need this to run borg on WSL (Windows Subsystem for Linux) or
 in systemd.nspawn containers on some architectures (e.g. ARM).
 Using this does not affect data safety, but might result in a more bursty
 write to disk behaviour (not continuously streaming to disk).
+.TP
+.B retryreadonly
+Retry opening a file without O_NOATIME if opening a file with O_NOATIME
+caused EROFS. You will need this to make archives from volume shadow copies
+in WSL1 (Windows Subsystem for Linux 1).
 .UNINDENT
```

Comment:
this is a generated file, do not edit / update / commit it (see line 1).

(all these files will be updated automatically at release time)